packages/hawtio/src/plugins/shared/operations/operation.ts

```diff
@@ -23,13 +25,21 @@ function addOperation(
   name: string,
   op: IJmxOperation,
 ): void {
-  const operation = new Operation(
-    name,
-    op.args.map(arg => new OperationArgument(arg.name, arg.type, arg.desc)),
-    op.desc,
-    op.ret,
-    op.canInvoke,
-  )
+  const args: OperationArgument[] = []
+  for (const arg of op.args) {
+    let argObj = arg
+    if (!isObject(arg)) {
+      try {
+        argObj = JSON.parse(arg as string)
+      } catch (error) {
+        log.error(`Cannot parse argument ${arg} in operation ${op.desc}`, error)
```

Comment:
Oftentimes `op.desc` doesn't provide enough information, so it's better to use `name` here.